Automattic/wp-calypso
# State: Introduce functionality for WP.org plugin lists

#### Changes proposed in this Pull Request

This PR introduces action creators, reducers, and selectors to mirror the existing `lib/plugins/wporg-data` flux store. The existing code that uses the old flux store will be ported in another PR.

Part of #24180.

#### Testing instructions

* Verify tests pass.
* If you're interested to manually test it, you can try:
  * Dispatching new actions for requesting and receiving data, then observing the reducer behavior
  * Hooking the action creators and selectors in a component and dispatching actions there, then testing selectors return value.
